inbox/events/microsoft/parse.py

```diff
@@ -287,72 +288,54 @@ def convert_msgraph_patterned_recurrence_to_ical_rrule(
     )
 
 
-class SynthetizedCanceledoccurrence:
+def synthetize_canceled_occurrence(
```

Comment:
spelling: synthesize

Reply:
LOL, I've been spelling this wrong all my life.